modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -949,9 +934,9 @@ private CompletableFuture<Void> startPartitionAndStartClient(
                     return false;
                 }
 
-                if (((Loza) raftMgr).isStarted(raftNodeId)) {
+                if (replicaMgr.isRaftClientStarted(raftNodeId)) {
```

Comment:
I think the whole if can be replaced by the method like replicaMgr.forcePeersUpdateIfNeeded(...) - again, it will encapsulate some raft logic to Replica layer.

Reply:
Will be implemented in IGNITE-22218